Makefile

```diff
@@ -133,8 +136,14 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	sed -i "" "s/namespace: splunk-operator/namespace: ${NAMESPACE}/g"  config/default/kustomization.yaml
+	sed -i "" "s/value: WATCH_NAMESPACE_VALUE/value: \"${WATCH_NAMESPACE}\"/g"  config/default/kustomization.yaml
```

Comment:
Does this sed format work on Linux? I think it may fail, as the -i options are different. Same comment for other places below.

Reply:
I thought I fixed it . sorry its for mac, I wanted to add for mac and linux , let me fix it